### DISCUSSION TITLE
.NET CORE

### DISCUSSION THREAD

**Discussion by bspeck74** [arkypita/LaserGRBL#1523] [Ideas]
Have you thought about move the lasergrbl into the .NET CORE.  this would allow more users to use the application at the same time consolidating the base.  This would allow future enhancements such as using mobile devices and wifi.
with the new features in .NET 6 and VS 2022 I think it would take the application to the next level.

**Comment by arkypita** [comment]
LaserGRBL user interface is totally written using winform.
Moving in .NET core has its advantage (maybe also multiplatform?) but require a lot of work,

**Comment by bspeck74** [comment]
Would you mind if I dl repo and take a look.  I have been a .NET developer
for almost 20 years.

**Reply by arkypita** [reply]
Sorry for late reply
You are welcome to fork and do your test, I will give you my support if you have any doubts about parts of the code.
